priv/demo/fake_operations_reports.yaml

```diff
@@ -0,0 +1,19 @@
+targets:
+  target1: c1f161e0-56b8-57cb-a28c-e5e581057b24
+
+
+reports:
+  "saptuneapplysolution@v1":
+    0:
+      target1: 
+        before: 
+          solution: ""
+        after: 
+          solution: "HANA"
+  "saptunechangesolution@v1":
+    0:
+      target1: 
+        before: 
+          solution: "HANA"
+        after: 
+          solution: "S4HANA-DBSERVER"
```

Comment:
This structure is meant to define what report each `target` is going to return for a given operation/step.

References to `target1`, `target2` will be resolved with relevant uuids when parsing the file. 
It is handy to keep the yaml tidy and less noisy by avoiding repetition of uuids